db/migrate/20180808030435_add_drug_gene_hits_to_amr_counts.rb

```diff
@@ -0,0 +1,5 @@
+class AddDrugGeneHitsToAmrCounts < ActiveRecord::Migration[5.1]
+  def change
+    add_column :amr_counts, :drug_gene_hits, :integer
```

Comment:
this is good for now until you get everything working. But before merging to master, we should consolidate all of these migrations into a single one. Probably inside create_amr_counts. You can rollback your local migrations that have already been applied using `rails db:rollback` multiple times. Then edit the file `db/migrate/20180726220853_create_amr_counts.rb` to add these additional columns. Then `rails db:migrate` again.
You should know that there was no need to do `rails generate migration` multiple times -- you could have done it ones and then edited the migration file to add more `add_column` statements

Reply:
Great, thanks for that suggestion !